### PR TITLE
Quorum Blockchain Service extension initial release

### PR DESCRIPTION
**Pull request by Null000**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

`az qbs`

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by Null000** [comment]
@microsoft-github-policy-service agree [company="{ConsenSys}"]

**Comment by Null000** [comment]
@microsoft-github-policy-service agree company="ConsenSys"

**Comment by yonzhan** [comment]
Quorum Blockchain Service

File: src/qbs/azext_qbs/commands.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        m.custom_command('list', 'get_invites')
+        m.custom_command('create', 'post_invites')
+        m.custom_command('revoke', 'delete_invites')
+        m.custom_command('validate', 'get_invites_valid')
+    with self.command_group('qbs transaction-node') as m:
+        m.custom_show_command('show', 'get_transaction_show')
+        m.custom_command('list', 'get_transaction_list')
+        m.custom_command('list-api-keys', 'get_transaction_list_api_keys')
+        m.custom_command('regenerate-api-keys', 'post_transaction_regenerate_api_keys')
+        m.custom_command('add', 'put_transaction')
```
Comment (kairu-ms) [inline src/qbs/azext_qbs/commands.py:31]: Please following this [guildline](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#command-naming-and-behavior) for command naming

**Comment by jamesstanleystewart** [comment]
Hi CLI team, can you please close this PR for the time being. Cheers

**Pull request closed without merging**